scipy/interpolate/fitpack2.py

```diff
@@ -172,8 +172,8 @@ def __init__(self, x, y, w=None, bbox=[None]*2, k=3, s=None,
             if (not np.isfinite(x).all() or not np.isfinite(y).all() or
                     not w_finite):
                 raise ValueError("x and y array must not contain NaNs or infs.")
-        if not all(diff(x) > 0.0):
-            raise ValueError('x must be strictly increasing')
+        if not all(diff(x) >= 0.0):
+            raise ValueError("x must be increasing")
```

Comment:
maybe `nondecreasing` is a better word here?

Reply:
I like this better as well.